airbyte-ci/connectors/connectors_qa/connectors_qa/consts.py

```diff
@@ -0,0 +1,23 @@
+# Copyright (c) 2023 Airbyte, Inc., all rights reserved.
+
+from base_images.python.bases import AirbytePythonConnectorBaseImage  # type: ignore
+
+CONNECTORS_QA_DOC_TEMPLATE_NAME = "connectors_qa.md.j2"
+DOCKER_HUB_PASSWORD_ENV_VAR_NAME = "DOCKER_HUB_PASSWORD"
+DOCKER_HUB_USERNAME_ENV_VAR_NAME = "DOCKER_HUB_USERNAME"
+DOCKER_INDEX = "docker.io"
+DOCKERFILE_NAME = "Dockerfile"
+DOCUMENTATION_STANDARDS_URL = "https://hackmd.io/Bz75cgATSbm7DjrAqgl4rw"
```

Comment:
Wow wow wow hold up, why are our standards in a 3rd party service and not a page on our very own docs site? Any specific reason? /cc @girarda @alafanechere

Reply:
No idea. This is the first time I see this page. I think this was hacked together by dev-rel two years ago, but that's only based off this [slack thread](https://airbytehq-team.slack.com/archives/C02UQ9MJ4GG/p1653835061244979).

Reply:
@natikgadzhi @girarda this doc was written by our previous documentation team. I believe it was drafted when I was working on the original `qa_checks.py` and never got checked in our main repo. I believe the doc is good enough to join our `Resources` section of `Contributing to Airbyte`. I can do it in a follow up PR.

<img width="288" alt="Screenshot 2024-02-16 at 08 09 30" src="https://github.com/airbytehq/airbyte/assets/5551758/dbed8285-34a3-466d-aff4-95c5b5b50e8e">